### PR TITLE
Generating `ids`

### LINKED ISSUE #59: Shorten the `id` property of `quizzes` and `questions`
- [ ] Find a consistent way to have shorter ids  
- [ ] Shouldn't be handmade ones
<br/>

>
> __PRIORITY__: MEDIUM
>

### PR DESCRIPTION
**Pull request by YacineSteeve**
⚠️ Depends on #62 👈🏾 (a little bit)

Resolves #59 

## The content

Here are two methods I suggest for getting `ids`:

* The first one simply uses the `uuid` Python module
* The second one uses the `nanoid` module (See [the doc](https://github.com/ai/nanoid#readme))

Both seems to be good solutions, but I would go for `nanoid`. This [article](https://blog.bitsrc.io/why-is-nanoid-replacing-uuid-1b5100e62ed2) show why.

Which one do you prefer ? Or does none of them fits our needs ?

### CODE REVIEW

File: backend/api/utils.py [hunk 1]
```diff
@@ -42,4 +44,9 @@ def get_sort_order(query_params: QueryDict, collection: Collection):
 
 
 def get_new_id() -> str:
```
Comment (chrichri17) [inline backend/api/utils.py:46]: How do you ensure that the IDs will be unique?

**Review by chrichri17 — COMMENTED** [review]
Let's discuss this a bit

File: backend/api/utils.py (hunk 1 above)
Comment (chrichri17) [reply backend/api/utils.py:46]: Why are you using both `nanoid` and `uuid` ? We can use `uuid` only right ? can you explain ?

Comment (YacineSteeve) [reply backend/api/utils.py:46]: Oh, it was just to show them two. I'll remove the one we don't keep.

Comment (YacineSteeve) [reply backend/api/utils.py:46]: As it is, I'm not absolutely sure it is unique. But the colliding rate is said to be quite low...
What I could do is to generate a new ID until the generated one is not already existing.

Comment (chrichri17) [reply backend/api/utils.py:46]: Hum ok i see. Is there a way to use the timestamp as well in the id ? Or while generating ? It might ensure unicity.

**Comment by chrichri17** [comment]
Sorry for the late review.. Holidays..

**Comment by chrichri17** [comment]
> Which one do you prefer ? Or does none of them fits our needs ?

Let's go with nanoid. But we might wwant to ensure unicity

**Review by chrichri17 — APPROVED** [review]

**Comment by YacineSteeve** [comment]
> Let's discuss this a bit

I come with another suggestion. Can you take a look at solutions __1 and 2__ of this [stackoverflow answer](https://stackoverflow.com/a/66366848/17404872) ?

**Comment by YacineSteeve** [comment]
> > Let's discuss this a bit
> 
> I come with another suggestion. 

I had some stuff here for you, @chrichri17

**Comment by YacineSteeve** [comment]
Too much outdated. I'll come back to it later.

**Pull request closed without merging**